sklearn/utils/_repr_html/params.py

```diff
@@ -0,0 +1,83 @@
+# Authors: The scikit-learn developers
+# SPDX-License-Identifier: BSD-3-Clause
+
+import html
+import reprlib
+from collections import UserDict
+
+from sklearn.utils._repr_html.base import ReprHTMLMixin
+
+
+def _params_html_repr(params):
```

Comment:
Let's add a small unit test by passing 2 parameters and making sure that they appear in the returned HTML template.

It should be really simple test. Maybe we could add in the test that we are checking for the string `"parameters-table"` because it is a CSS rule and we don't want people to modify it without knowing. Maybe the same for the string `"estimator-table"`

Reply:
Somehow this unit test should replace the one that we have in `test_estimator.py`.